snapshot-labs/sx-monorepo
# fix: fix invalid default network on space creation

### Summary

Fix invalid default selected network on space creation

Closes: #34

This issue is due to the introduction of offchain networks, which are not shown on the space creation step, making the default selected network `s`.

This PR is a temporary fix, and is filtering out offchain networks, which are not used on space creation. (filter will be removed once we can create offchain spaces on sx)

### How to test

Before the fix

1. Go to the create space page (http://localhost:8080/#/create)
2. Fill in a name (mininum requirement to move to next step)
3. Click on NEXT on the page bottom
4. It move to the NETWORK step
5. No network is selected by default

After the fix

1. Go to the create space page (http://localhost:8080/#/create)
2. Fill in a name (mininum requirement to move to next step)
3. Click on NEXT on the page bottom
4. It move to the NETWORK step
5. Ethereum network should be selected by default
